charts/ingress-nginx/values.yaml

```
@@ -85,6 +85,8 @@ controller:
   enableTopologyAwareRouting: false
   # -- This configuration disable Nginx Controller Leader Election
   disableLeaderElection: false
+  # -- Duration a leader election is valid before it's getting re-elected. If not set, the default value defined on controller is 30.
+  electionTTL: ""
```

I'm suggesting to modify the helm-docs comment to change the type of `electionTTL` to `int` and to set its default value to `null`. Leaving the default value to empty string, users could be confused to use golang duration strings which do not seem to be supported by your new flag.

After applying the change, please re-run helm-docs to make it reflect in the values table in the Readme.

```suggestion
  # -- (int) Duration a leader election is valid before it's getting re-elected. If not set, the default value defined on controller is 30.
  electionTTL:
```